\chapter{Discussion}

It is clear that more robust statistical analysis is necessary in this field. The chapter on expanding the UnIFrac toolbox highlighted examples of misuse of unweighted UniFrac in papers published in Cell and Nature. One claimed to find differences in the gut microbiome of mice modelling autism spectrum disorder, compared to healthy controls. The other claimed to find differences in the gut microbiome of humanized mice fed a more traditional fibrous diet compared to mice compared to mice fed a diet similar in composition to the Western diet. These studies had small sample sizes (n = 20 and 10 respectively), used unweighted UniFrac (which we have shown to be unreliable), and had a low amount of variance explained by the principal components axes (14\% on PC1).

The chapter about nonalcoholic fatty liver disease (NAFLD) showcased five studies which all claimed to have found a difference in the gut microbiome of patients with nonalcoholic fatty liver disease compared to healthy controls, but with almost non-overlapping results (Fig.~\ref{nafld_fig1}). Some of the variation can be explained by differences in sequencing platform (Roche 454 vs. Illumina MiSeq). More variation can be explained by the variable region of the 16S rRNA gene chosen for sequencing - one study used V1-2, one study used V3, one study used V4, and the other two studies did not report which variable region was used. Three out of five studies used healthy controls with a lower BMI than the NAFLD group, such that differences due to level of obesity could not be distinguished from differences due to NAFLD. Lastly, only one of the studies performed a multiple test correction, so most of the results could not be distinguished from false positives.

recommendations
- compositional data analysis
- look at your data (variance explained, etc.)
- ways to make inferences on data that are not p-value based (like our effect size stuff)

nature statistics biology

summary/conclusion
- we did some methods development and applied it to nafld
- found that many studies in this field cannot be replicated
- made recommendations

field needs some standards like clinical genomics (other fields?)
GWAS studies - reproducibility/batch effects

microbiome standardized way too early, before they knew what they were looking at
- the human microbiome project
- done by ecologists ?
 â€” ecological diversity (one of hte HMP paper in supplementary data, found that diversity indices track how deeply you sequence and tell you nothing else)

- learned as they went along, but were under a tight timeline